package.json

```diff
@@ -51,6 +53,12 @@
     "eslint-plugin-react": "7.21.5",
     "eslint-plugin-react-hooks": "1.7.0",
     "eslint-plugin-testing-library": "3.3.1",
-    "prettier": "^2.2.1"
+    "husky": ">=6",
+    "lint-staged": ">=10",
+    "prettier": "^2.2.1",
+    "pretty-quick": "^3.1.0"
+  },
+  "lint-staged": {
+    "*.{tsx,ts}": "pretty-quick --staged"
```

Comment:
This is just running prettier? Why not run the lint script? Prettier has been integrated via the eslint plugin...